controllers/rcj_soccer_referee_supervisor/referee/consts.py

```diff
@@ -62,6 +62,26 @@ class Team(Enum):
 YELLOW_PENALTY_AREA = (-0.59, -0.35, 0.35)
 BLUE_PENALTY_AREA = (0.59, -0.35, 0.35)
 
+MAX_EVENT_MESSAGES_IN_QUEUE = 10
+
+
+class LabelIds:
+    BLUE_SCORE = 0
+    YELLOW_SCORE = 1
+    TIME = 2
+
+    # Event messages - based on MAX_EVENT_MESSAGES_IN_QUEUE
+    EVENT_MESSAGE_1 = 10
+    EVENT_MESSAGE_2 = 11
+    EVENT_MESSAGE_3 = 12
+    EVENT_MESSAGE_4 = 13
+    EVENT_MESSAGE_5 = 14
+    EVENT_MESSAGE_6 = 15
+    EVENT_MESSAGE_7 = 16
+    EVENT_MESSAGE_8 = 17
+    EVENT_MESSAGE_9 = 18
+    EVENT_MESSAGE_10 = 19
+
 
 class GameEvents(Enum):
```

Comment:
@mrshu I think we can get rid of this Enum, since we are not using its features and make it similar to LabelIds class

Reply:
@Adman I guess `Enum` gives us things like `GameEvents['MATCH_START']` for free, so I don't know. You are probably right though, there is a very good chance we won't use any of these features.